### DISCUSSION TITLE
Webradio doesn´t play  .m3u Streams

### DISCUSSION THREAD

**Discussion by Matzeman** [rern/rAudio#87] [Fixed Issues]
Version 20200105 is installed. I have the following problem with the web radio. Added a new station: 
http://mp3-live.swr3.de/swr3_m.m3u
But the stream does not play. No matter which .m3u file I read in, none is played.

Is this a bug?

**Comment by Sagnasty** [comment]
@Matzeman 

This may help you: [https://github.com/y20k/transistor/issues/138](url)

I noticed that if I open your web radio stream using VLC media player, the link was translated to [http://swr-swr3-live.cast.addradio.de/swr/swr3/live/mp3/128/stream.mp3](url)

The translated link will then play in rAudio 1 20210105

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by Matzeman** [comment]
@Sagnasty,

Thanks for the info. It works. I thought I could only enter .pls or .m3u files.
